ui/src/dashboards/components/TemplateControlBar.js

```diff
@@ -5,26 +5,48 @@ import Dropdown from 'shared/components/Dropdown'
 
 import omit from 'lodash/omit'
 
+const pixelsPerCharacter = 9
+const minTempVarDropdownWidth = 146
+
 const TemplateControlBar = ({
   templates,
   onSelectTemplate,
   onOpenTemplateManager,
   isOpen,
-}) => (
+}) =>
   <div className={classnames('template-control-bar', {show: isOpen})}>
     <div className="template-control--container">
       <div className="template-control--controls">
         {templates.length
           ? templates.map(({id, values, tempVar}) => {
               const items = values.map(value => ({...value, text: value.value}))
+              const itemValues = values.map(value => value.value)
               const selectedItem = items.find(item => item.selected) || items[0]
               const selectedText = selectedItem && selectedItem.text
+              let customDropdownWidth = 0
+              if (itemValues.length > 1) {
+                const longest = itemValues.reduce(function(a, b) {
+                  return a.length > b.length ? a : b
+                })
```

Comment:
This should be written as:
```javascript
const longest = itemValues.reduce((a, b) => a.length > b.length ? a : b)
```
The reason for that is because we don't use non-arrow functions anywhere else in the project.